src/schemas/resetToken.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const ResetTokenSchema = new mongoose.Schema({
+  resetToken: {
+    type: String,
+    required: true,
+  },
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User', 
+    required: true,
+  },
+  createdAt: {
+    type: Date,
+    required: true,
+  },
+});
+
+const ResetToken = mongoose.model('ResetToken', ResetTokenSchema);
+
```

Comment:
## Future Suggestions
It would be better to have a method to automatically delete expired tokens from the database.